encryptor/config_loader/consul/storage.go

```diff
@@ -0,0 +1,157 @@
+package consul
+
+import (
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/cossacklabs/acra/encryptor"
+	"github.com/hashicorp/consul/api"
+	log "github.com/sirupsen/logrus"
+)
+
+// StorageCreator implement config_loader.EncryptorConfigStorage via Hashicorp Consul Backend
+type StorageCreator struct{}
+
+// StorageConfigured check weather CLI flag for Consul using was provided
+func (s StorageCreator) StorageConfigured(flags *flag.FlagSet, prefix string) bool {
```

Comment:
Lets use some question in the method name to show that it returns boolean? IsConfigured, DoesConfigured, etc.